flume/doc/operation/hdfs_kerberos_authentication.md

```diff
@@ -0,0 +1,67 @@
+# Enabling Kerberos SPNEGO authentication in HDFS
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-connectors/commit/af19dc77924e4d15f2d0dfc86ac8d0ce2cf30c63